### ISSUE TITLE
Total Time to Train COCO by Default, for MaskRCNN?

### ISSUE THREAD

**Issue by kuonangzhe** [dmlc/gluon-cv#381]
Thanks for the great work to support MaskRCNN. 

I am trying to train MaskRCNN for COCO instance segmentation, using the default settings. While the only difference is that I changed coco2017 to coco2014, so I can train on existing coco 2014 data. 

The speed is pretty low for training one epoch, nearly 6 - 8 hours. I used default train_mask_rcnn.py, while using 8 1080Ti graphics cards. 

Other Settings:
mxnet: 1.3.0. 
cuda: 8.0
ubuntu: 16.04
python: 3.5

Is the training speed normal? Since the default epoch was 26, which might cost over one week or so. Thanks in advance!

**Comment by zhreshold** [comment]
COCO is quite large dataset, it's 3.5 hours  per epoch on our machine. You can of course use less than 26 epochs, for example, 13 epochs and set learning rate scheduler accordingly.
The result will be slightly worse, but still acceptable.

**Comment by kuonangzhe** [comment]
Thanks for the help! I got it~
